.luarc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "diagnostics.disable": ["undefined-global", "deprecated", "lowercase-global"]
```

Comment:
Disable some diagnostics for now. If a third party documentation environment is available on the Lua Language Server, we can change this.